DavidGonzalezZumaya/Collections/src/JavaMap/JavaMap.java

```diff
@@ -0,0 +1,36 @@
+package JavaMap;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class JavaMap {
+
+    public static void main(String []argh)
+    {
+        Scanner in = new Scanner(System.in);
+        Map<String, Integer> phoneDirectory = new HashMap<>();
+
+        int directorySize = in.nextInt();
+        in.nextLine();
+
+        for(int index=0; index < directorySize; index++){
```

Comment:
Remember all binary operators and = except . should be separated from their operands by spaces. Here you can find more information https://www.oracle.com/java/technologies/javase/codeconventions-whitespace.html